app/src/main/java/com/sun/makeupwindow/data/repository/ProductRepository.kt

```diff
@@ -0,0 +1,37 @@
+package com.sun.makeupwindow.data.repository
+
+import com.sun.makeupwindow.data.model.Product
+import com.sun.makeupwindow.data.source.ProductDataSource
+import com.sun.makeupwindow.data.source.utils.OnDataLoadCallback
+
+class ProductRepository private constructor(
+    private val local: ProductDataSource.Local,
+    private val remote: ProductDataSource.Remote
+) : ProductDataSource.Local, ProductDataSource.Remote {
+
+    override fun getProduct(callback: OnDataLoadCallback<List<Product>>) = local.getProduct(callback)
+
+    override fun getItemProduct(idproduct: Int, callback: OnDataLoadCallback<Product>) {
+        return local.getItemProduct(idproduct, callback)
+    }
```

Comment:
```suggestion
        override fun getItemProduct(idProduct: Int, callback: OnDataLoadCallback<Product>) = 
            local.getItemProduct(idProduct, callback)
```
chỗ nào tương tự thì sửa đi em, sửa mỗi chỗ review thôi vậy.